### PR TITLE
Update `StructureVisualizer`

### PR DESCRIPTION
**Pull request by vachillo**
- [x] I have read and agree to the contributing guidelines for [submitting new pull requests](https://github.com/griptape-ai/griptape?tab=readme-ov-file#submitting-pull-requests).

## Describe your changes
properly fixes this

### CODE REVIEW

File: tests/unit/utils/test_structure_visualizer.py [hunk 1]
```diff
@@ -27,9 +29,11 @@ def test_pipeline(self):
         visualizer = StructureVisualizer(pipeline)
         result = visualizer.to_url()
 
+        print(result)
```
Comment (collindutter) [inline tests/unit/utils/test_structure_visualizer.py:32]: Remove print

File: tests/unit/utils/test_structure_visualizer.py [hunk 2]
```diff
@@ -11,7 +11,9 @@ def test_agent(self):
         visualizer = StructureVisualizer(agent)
         result = visualizer.to_url()
 
-        assert result == "https://mermaid.ink/svg/Z3JhcGggVEQ7CgkndGFzazEnOw=="
+        print(result)
```
Comment (collindutter) [inline tests/unit/utils/test_structure_visualizer.py:14]: Remove print

**Review by collindutter — CHANGES_REQUESTED** [review]
Can you provide more context to what this fixes?

**Comment by vachillo** [comment]
> Can you provide more context to what this fixes?

#894 but a better fix. this creates hashed mermaid-safe ids but correctly prints the task name (without the single quotes)

**Review by collindutter — APPROVED** [review]

**Pull request merged**